doc/man3/CRYPTO_THREAD_run_once.pod

```diff
@@ -74,6 +76,14 @@ operations are supported on the specific platform. Because of this, if a
 variable is modified by CRYPTO_atomic_add() then CRYPTO_atomic_add() must
 be the only way that the variable is modified.
 
+=item *
+
+CRYPTO_atomic_read() atomically reads B<val> and returns the result of
+the operation in B<ret>. B<lock> will be locked, unless atomic operations
+are supported on the specific platform. Because of this, if a variable
+is read by CRYPTO_atomic_read() then CRYPTO_atomic_read() must
```

Comment:
shouldn't that second read mention be atomic add?

Reply:
Yeah :(

Reply:
And removed.